W-17772543-SFTP-2-4-2-Write-operation-custom-mode-modifications (#2788)
mulesoft/docs-connectors: sftp/2.4/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc

=== Override Standard Write Strategy

> If your *File Write* operation overrides instead of appending when using the `APPEND` mode, use the `CUSTOM` write strategy along with the `CUSTOM_APPEND` mode only as it isn't applicable to the other modes. With this strategy, you can manually specify the offset for the server to write the file in `APPEND` mode, particularly only for older SFTP servers that don't directly support write modes. Use this strategy only if the `STANDARD` strategy results in errors during file writing in `APPEND` mode and doesn’t offer concurrency. Additionally, you can adjust the buffer size in the *Buffer Size for Write Strategy* field. 

Reviewer: ```suggestion
If your *File Write* operation overrides the `max idle` configuration instead of appending it when using the `APPEND` mode, use the `CUSTOM` write strategy along with the `CUSTOM_APPEND` mode. This strategy isn't applicable to the other modes. With this strategy, you can manually specify the offset for the server to write the file in `APPEND` mode. This strategy is useful for older SFTP servers that don't directly support the `WRITE` mode. Use this strategy only if the `STANDARD` strategy results in errors during file writing in `APPEND` mode and doesn’t offer concurrency. Additionally, adjust the buffer size in the *Buffer Size for Write Strategy* field. 
```

Reviewer: Please, review and ensure that my changes are appropriate.
